.drone.yml

```diff
@@ -55,6 +55,8 @@ pipeline:
       - npm_registry
       - harbor_sign_key
       - harbor_sign_key_id
+      - use_internal_tdnf
```

Comment:
I don't think  we need the flag `use_internal_tdnf` how about we just check if the `harbor_tdnf_repo` is set  ` ! -z $HARBOR_TDNF_REPO ]`